### PR TITLE
Feat/server/command connect nbr

### LINKED ISSUE #47: Implement the `connect_nbr` command (server-side)

### PR DESCRIPTION
**Pull request by Eviber**
Fixes #47.

This branch is based on `client-buffer`, and should therefore be merged after #53.

This is just a proposition, there are other ways to do this.

### CODE REVIEW

**Comment by Eviber** [comment]
(Just watch the last commit to see the changes.)

**Comment by Eviber** [comment]
ok, it works but it feels sloppy, any ideas?

File: crates/server/src/state/mod.rs [hunk 1]
```diff
@@ -63,12 +63,22 @@ impl Command {
...
+            _ => Response::Ok,
+        }
+    }
 }
 
 /// A response that can be sent back to a player.
 pub enum Response {
     /// The string `"ok"`.
     Ok,
+    ConnectNbr(u32),
```
Comment (Ragarnoy) [inline crates/server/src/state/mod.rs:81]: MISSING DOC FUCK YOU

**Review by Ragarnoy — CHANGES_REQUESTED** [review]
DOC

**Review by nils-mathieu — APPROVED** [review]

**Review by Ragarnoy — APPROVED** [review]

**Pull request merged**